src/util.js

```diff
@@ -1470,7 +1470,7 @@ export const importText = (itemsRanked, inputText) => {
           importCursor.pop()
         }
       }
-    }, { decodeEntities: true })
```

Comment:
This is definitely part of the issue. We may have to manually encode characters like `&` since we do not want `<` to be encoded. Maybe [htmlparser2](https://github.com/fb55/htmlparser2) has more options.